### PR TITLE
GTC DLI workflow docs

### PR DESCRIPTION
**Pull request by peterd-NV**
## Summary
Add docs for GTC DLI course GR1 sequential static manipulation workflow

### CODE REVIEW

File: docs/pages/example_workflows/locomanipulation/step_4_evaluation.rst [hunk 1]
```diff
@@ -16,10 +16,6 @@ Once inside the container, set the dataset and models directories.
     export DATASET_DIR=/datasets/isaaclab_arena/locomanipulation_tutorial
     export MODELS_DIR=/models/isaaclab_arena/locomanipulation_tutorial
 
-.. todo::
-
-   Verifying Arena-GR00T on Blackwell architecture.
-
```
Comment (alexmillane) [inline docs/pages/example_workflows/locomanipulation/step_4_evaluation.rst:22]: 🥳

File: docs/pages/example_workflows/sequential_static_manipulation/index.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+   * - **Tags**
+     - Table-top manipulation
+   * - **Skills**
+     - Pick & place, Close door
+   * - **Embodiment**
+     - Fourier GR1T2 (54 DOF humanoid)
+   * - **Interop**
+     - Isaac Lab Teleop, Isaac Lab Mimic
+   * - **Scene**
+     - Kitchen environment
```
Comment (alexmillane) [inline docs/pages/example_workflows/sequential_static_manipulation/index.rst]: Maybe we better put Light Wheel Robocasa Kitchen to attribute LightWheel.

File: docs/pages/example_workflows/sequential_static_manipulation/index.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+   * - **Objects**
+     - Refrigerator (articulated object)
```
Comment (alexmillane) [inline docs/pages/example_workflows/sequential_static_manipulation/index.rst]: - object to place in refrigerator (rigid object)

File: docs/pages/example_workflows/sequential_static_manipulation/step_3_data_generation.rst [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+     --enable_pinocchio \
+     --mimic \
+     put_item_in_fridge_and_close_door \
+     --object ranch_dressing_bottle \
+     --embodiment gr1_pink
+
+Follow the instructions described on the CLI to mark subtask boundaries:
+
+Left Arm:
+1. **Pick & place completed:** Robot has completed the pick and place task
```
Comment (alexmillane) [inline docs/pages/example_workflows/sequential_static_manipulation/step_3_data_generation.rst:67]: Perhaps it's made clear in the CLI, but I'm wondering whether we should specify where the user should press the key here. Does this mean that you press the button once the door is closed? How does that interact with the reset.

File: docs/pages/example_workflows/sequential_static_manipulation/step_3_data_generation.rst [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+Data generation takes 30-60 minutes depending on hardware.
+If you want to visualize the data generation process, remove ``--headless``
+to visualize data generation.
+
+
+Step 3: Validate Generated Data (Optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to validation the generated dataset, you can replay the generated data
```
Comment (alexmillane) [inline docs/pages/example_workflows/sequential_static_manipulation/step_3_data_generation.rst]: nit: in order to validate

File: docs/pages/example_workflows/sequential_static_manipulation/step_4_policy_training.rst [hunk 6]
```diff
@@ -0,0 +1,268 @@
...
+
+Step 2: Verify Generated LeRobot Dataset
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To verify the generated LeRobot dataset, you first need to compute statistics information about the dataset.
+This can be done by running the following command:
+
+.. code-block:: bash
+
+   python submodules/Isaac-GR00T/gr00t/data/stats.py $DATASET_DIR/ranch_bottle_into_fridge/ranch_bottle_into_fridge_generated_100/lerobot gr1_joint
```
Comment (alexmillane) [inline docs/pages/example_workflows/sequential_static_manipulation/step_4_policy_training.rst:105]: I notice that the stats command alters some files. Does this have an affect on training? I wasn't running this when training things in Zurich. I'm wondering if that could have affected the performance?

File: docs/pages/example_workflows/sequential_static_manipulation/step_4_policy_training.rst [hunk 7]
```diff
@@ -0,0 +1,268 @@
...
+
+.. todo::
+   There is a bug in ISAAC-GR00T submodule gr00t/data/stats.py. PR is submmited and in review from Gear.
+   Usele gpu finetune command in the last step for now.
```
Comment (alexmillane) [inline docs/pages/example_workflows/sequential_static_manipulation/step_4_policy_training.rst]: Use finetune command?

File: docs/pages/example_workflows/sequential_static_manipulation/step_4_policy_training.rst [hunk 8]
```diff
@@ -0,0 +1,268 @@
...
+
+      To post-train the policy, run the following command
+
+      .. code-block:: bash
+
+         python -m torch.distributed.run --nproc_per_node=8 --standalone submodules/Isaac-GR00T/gr00t/experiment/launch_finetune.py \
+         --dataset_path=$DATASET_DIR/arena_gr1_manipulation_dataset_generated/lerobot \
+         --output_dir=$MODELS_DIR \
+         --modality_config_path=isaaclab_arena_gr00t/embodiments/gr1/gr1_arms_only_data_config.py \
+         --global_batch_size=24 \
```
Comment (alexmillane) [inline docs/pages/example_workflows/sequential_static_manipulation/step_4_policy_training.rst]: shall we go with 96? In my testing this results in a lower training error, and doesn't affect the training time.

File: docs/pages/example_workflows/sequential_static_manipulation/step_5_evaluation.rst [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+
+
+Test the policy in a single environment with visualization via the GUI run:
+
+.. code-block:: bash
+
+   python isaaclab_arena/evaluation/policy_runner.py \
+     --policy_type isaaclab_arena_gr00t.policy.gr00t_closedloop_policy.Gr00tClosedloopPolicy \
+     --policy_config_yaml_path isaaclab_arena_gr00t/policy/config/gr1_manip_ranch_bottle_gr00t_closedloop_config.yaml \
+     --num_steps 2000 \
```
Comment (alexmillane) [inline docs/pages/example_workflows/sequential_static_manipulation/step_5_evaluation.rst:77]: To me it makes more sense that we run for a number of episodes. I'm adding that here: #383 

What do you think?

File: docs/pages/example_workflows/sequential_static_manipulation/index.rst (hunk 2 above)
Comment (xyao-nv) [reply docs/pages/example_workflows/sequential_static_manipulation/index.rst]: done

File: docs/pages/example_workflows/sequential_static_manipulation/index.rst (hunk 3 above)
Comment (xyao-nv) [reply docs/pages/example_workflows/sequential_static_manipulation/index.rst]: done

File: docs/pages/example_workflows/sequential_static_manipulation/step_3_data_generation.rst (hunk 5 above)
Comment (xyao-nv) [reply docs/pages/example_workflows/sequential_static_manipulation/step_3_data_generation.rst]: done

File: docs/pages/example_workflows/sequential_static_manipulation/step_4_policy_training.rst (hunk 6 above)
Comment (xyao-nv) [reply docs/pages/example_workflows/sequential_static_manipulation/step_4_policy_training.rst:105]: stats.py added stats.json & rel_stats.json which I think it's used in dataloader for normalization. It's included in training but not replaying. I don't think it's a bug from stats ops itself, as it mostly copies from n1.5.

File: docs/pages/example_workflows/sequential_static_manipulation/step_4_policy_training.rst (hunk 7 above)
Comment (xyao-nv) [reply docs/pages/example_workflows/sequential_static_manipulation/step_4_policy_training.rst]: done

File: docs/pages/example_workflows/sequential_static_manipulation/step_4_policy_training.rst (hunk 8 above)
Comment (xyao-nv) [reply docs/pages/example_workflows/sequential_static_manipulation/step_4_policy_training.rst]: good catch. It was trained with 96 but I forgot to update the doc here

File: docs/pages/example_workflows/sequential_static_manipulation/step_5_evaluation.rst (hunk 9 above)
Comment (xyao-nv) [reply docs/pages/example_workflows/sequential_static_manipulation/step_5_evaluation.rst:77]: I think it's gonna be a great idea once you merge it. It looks like it has issue for recording metrics yesterday I ran it.

File: docs/pages/example_workflows/sequential_static_manipulation/step_3_data_generation.rst (hunk 4 above)
Comment (peterd-NV) [reply docs/pages/example_workflows/sequential_static_manipulation/step_3_data_generation.rst:67]: Adding this to docs
